src/mlpack/methods/ann/layer/mean_pooling.hpp

```diff
@@ -37,7 +37,7 @@ class MeanPooling
   MeanPooling();
 
   /**
-   * Create the MeanPooling object using the specified number of units.
+   * Create the MeanPooling object using the specified parameter.
```

Comment:
```suggestion
   * Create the MeanPooling object using the specified parameters.
```
(Same as last suggestion; up to you if you want to take it.)